framework/tests/nontopdefault/NTDTest.java

```diff
@@ -0,0 +1,45 @@
+import tests.nontopdefault.qual.*;
+
+/* Hierarchy:
```

Comment:
Would this hierarchy be better with the NTDChecker? Or as package-doc with the qualifiers?
Like this, it is unlikely to be kept in sync with the defaults specified with the qualifiers.